app/questionnaire/questionnaire_schema.py

```diff
@@ -154,19 +168,24 @@ def get_questions(self, question_id):
 
     def group_has_questions(self, group_id):
         for block in self.get_group(group_id)['blocks']:
-            if block['type'] == 'Question':
+            if QuestionnaireSchema.is_question_block_type(block['type']):
                 return True
         return False
 
     @staticmethod
-    def get_visible_list_blocks_for_section(section):
-        list_collector_blocks = []
-        for group in section['groups']:
-            for block in group['blocks']:
-                hidden = block.get('hide_on_section_summary', False)
-                if block['type'] == 'ListCollector' and not hidden:
-                    list_collector_blocks.append(block)
-        return list_collector_blocks
+    def get_list_blocks_for_section(section):
+        visible_list_collector_blocks = []
+        hidden_list_collector_blocks = []
+
+        for block in QuestionnaireSchema.get_blocks_for_section(section):
+            hidden = block.get('hide_on_section_summary', False)
```

Comment:
`hide_on_section_summary` should now be `show_on_section_summary`

Reply:
Validator uses hide_on_section_summary at the moment on v3?

Reply:
Yep - I'm actually going to make the change to validator and this block in my or to make this consistent with show_on_section_summary that's being introduced there

Reply:
So I'll dismiss this comment